clients/osv/src/main/kotlin/OsvClient.kt

```diff
@@ -0,0 +1,129 @@
+/*
+ * Copyright (C) 2022 The ORT Project Authors (see <https://github.com/oss-review-toolkit/ort/blob/main/NOTICE>)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+package org.ossreviewtoolkit.clients.osv
+
+import java.io.IOException
+import java.util.concurrent.ConcurrentLinkedQueue
+import java.util.concurrent.CountDownLatch
+import java.util.concurrent.atomic.AtomicReference
+
+import okhttp3.OkHttpClient
+
+import retrofit2.Call
+import retrofit2.Callback
+import retrofit2.Response
+
+/**
+ * This class wraps the OSV Rest API client in order to make its use simpler and less error-prone.
+ */
+class OsvClient(serverUrl: String? = null, httpClient: OkHttpClient? = null) {
```

Comment:
As you're renaming this class, I believe a better name would be `OsvServiceWrapper`. It's inline with the intention and the already existing class docs.

Reply:
IIUC, per definition the code that wraps the service *is* the client, so I'd prefer to keep the shorter (and more commonly used) "client".

Reply:
> IIUC, per definition the code that wraps the service is the client,

I see, so what is the code that _uses_, not _wraps_ the service then?
To me it seems that client would better fit to the code using that wrapper. 

> so I'd prefer to keep the shorter (and more commonly used) "client".

ok, normally I don't care too much, in this case I would appreciate if the compromise was made towards my preference.

Reply:
I've checked the code base again, and OSV seems to currently be the only client library project that providers a higher-level "wrapper" around the low-level `Service` interface. In my thinking that "wrapper" is something between the service and the (advisor) client, so going for "ServiceWrapper" here is fine with me.

Reply:
Or actually, thinking about it again, we could also keep `OsvServiceWrapper` at `OsvClient` to keep thing simple, as you prefer.

Reply:
I'd prefer to keep `OsvServiceWrapper` as it is right now.